app/controllers/gi_controller.rb

```diff
@@ -11,6 +11,8 @@ def client
   end
 
   def scrubbed_params
-    params.except(:action, :controller, :format)
+    params.except(:action, :controller, :format).transform_values do |v|
+      v.encode!('UTF-8', invalid: :replace, undef: :replace, replace: '')
```

Comment:
hmmmm, but wouldn't it still be best to raise a helpful error and perhaps log the actual query so we can track down precisely what the problem is? Replacing user supplied params is best practice, we don't really do it elsewhere. In general if some invalid parameter is provided we raise a helpful error to indicate what caused it. If you still feel strongly though, could we at least make sure we're logging the params as a warning or info so we can more easily dig into what causes this?

NOTE: if we do log to sentry, be sure to scrub out '%'